src/services/jwtService.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken');
+
+require('dotenv').config();
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY);
+
+  return token;
+};
+
+const verify = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const signRefresh = (user) => {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY);
```

Comment:
Consider adding an `expiresIn` option to the `jwt.sign` method for the refresh token as well, if applicable. This can help manage token lifecycle more effectively.